backend/src/main/java/com/webappsbusters/packagemanagement/controller/ParcelController.java

```diff
@@ -0,0 +1,28 @@
+package com.webappsbusters.packagemanagement.controller;
+
+import com.webappsbusters.packagemanagement.domain.ParcelDto;
+import com.webappsbusters.packagemanagement.exceptions.ParcelNotFoundException;
+import com.webappsbusters.packagemanagement.mapper.ParcelMapper;
+import com.webappsbusters.packagemanagement.service.ParcelService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/v1")
+public class ParcelController {
+
+    private final ParcelService parcelService;
+    private final ParcelMapper parcelMapper;
+
+    @Autowired
+    public ParcelController(ParcelService parcelService, ParcelMapper parcelMapper) {
+        this.parcelService = parcelService;
+        this.parcelMapper = parcelMapper;
+    }
+
+    @GetMapping(value = "parcels/{parcelId}")
```

Comment:
I think it should be /packages https://theflyingkeyboard.atlassian.net/browse/PAC-28, but we can discus

Reply:
Together with the Team we decided to stay with 'parcel' instead of 'package', so I changed it in other places in the project.